src/heist_path/HeistPathB.py

```
@@ -139,35 +138,40 @@ def goto_lg1_skip(self):
-            self.clear_current_combat_spec()
+        if self.wait_ocr(x=0.60, y=0.52, to_x=0.70, to_y=0.57, match=re.compile("开门"), time_out=1.14):
             self.sleep(0.10)
             self.send_key("f", down_time=0.10)
             self.sleep(0.10)
-            self.send_key("f", down_time=0.10)
+        elif self.find_interac():
+            self.clear_current_combat()
             self.sleep(0.10)
-            self.send_key("f", down_time=0.10)
+            isOpenDoor = False
+            while isOpenDoor == False:
```

Ruff (0.15.14)

[error] Avoid equality comparisons to `False`; use `not isOpenDoor:` for false checks

Replace with `not isOpenDoor`

(E712)